packages/salesforcedx-vscode-apex/package.nls.json

```diff
@@ -10,10 +10,11 @@
   "java_home_description": "Specifies the folder path to the Java 8 or Java 11 runtime used to launch the Apex Language Server (for example, /Library/Java/JavaVirtualMachines/jdk1.8.0_131.jdk/Contents/Home).",
   "java_memory_description": "Specifies the amount of memory allocation to the Apex Language Server in MB or null to use the default value.",
   "apex_semantic_errors_description": "Allow Apex Language Server to surface semantic errors",
+  "apex_completion_statistics_description": "List statistics for Apex Completions",
```

Comment:
@ekapner If I can, I want to hide this from customers. In the event that I cannot, does this wording look okay to you?

Reply:
@sfsholden - Lists the statistics for Apex completions 

lower case "completions"

Regarding the memory description, do we know the default value, or does it differ?

Reply:
@ekapner It looks like the value will differ for the java memory, as it is calculated at runtime and based on system configuration. This has some good info: https://stackoverflow.com/questions/4667483/how-is-the-default-max-java-heap-size-determined

Reply:
Thanks. I would edit the description to say (although not highlighted in green):
"...Apex Language Server in MB, or null to use the system default value."